src/mono/mono.proj

```diff
@@ -149,15 +149,15 @@
   <!-- Sets up emscripten if you don't have the EMSDK_PATH env variable set -->
   <Target Name="ProvisionEmscripten"
```

Comment:
Unrelated: why is emscripten provisioning in mono.proj instead of wasm.proj?

Reply:
@radekdoulik would know better. My guess is that it just happens to be there, and we can move it to wasm.proj .